firebase-installations/src/main/java/com/google/firebase/installations/local/PersistedFid.java

```diff
@@ -21,37 +21,38 @@
 import com.google.firebase.FirebaseApp;
 
 /**
- * A layer that locally caches a few Firebase Installation attributes on top the Firebase
- * Installation backend API.
+ * A layer that locally persists a few Firebase Installation attributes on top the Firebase
+ * Installation API.
  */
-public class FiidCache {
-  // Status of each cache entry
-  // NOTE: never change the ordinal of the enum values because the enum values are stored in cache
+public class PersistedFid {
+  // Status of each presisted fid entry
+  // NOTE: never change the ordinal of the enum values because the enum values are stored in shared
+  // prefs
   // as their ordinal numbers.
-  public enum CacheStatus {
-    // Cache entry is synced to Firebase backend
+  public enum PersistedStatus {
+    // Persisted Fid entry is synced to Firebase backend
     REGISTERED,
-    // Cache entry is waiting for Firebase backend response or internal network retry
+    // Persisted Fid entry is waiting for Firebase backend response or internal network retry
     UNREGISTERED,
```

Comment:
Don't we need a status "IN_PROGRESS" or "PENDING" for when the SDK is waiting for a request to the FIS server to complete?
Without that, you might have race conditions, where one call to #getId triggers a request to FIS and a second call to #getId triggers another request. The first one succeeds, the second one returns an error...